keps/sig-apps/2214-indexed-job/README.md

```diff
@@ -639,36 +656,36 @@ the existing API objects?**
   - Estimated increase in size: new annotation of about 50 bytes and hostname
     which includes the index.
 
-* **Will enabling / using this feature result in increasing time taken by any 
-operations covered by [existing SLIs/SLOs]?**
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
 
   No.
```

Comment:
I'm not sure this is the correct answer, the extra processing coming from indexed job seems to be one that might potentially affect non-index job processing. As I've mentioned before would be nice to measure the difference and specify the expectations differently for different types of jobs.

Reply:
It is the correct answer in the sense that Job is not covered by an SLO. @wojtek-t, any recommendation?

Reply:
Yeah - in the sense of the SLIs/SLOs we're documenting officially it's correct.

The recommendation would be to create a new SLI/SLO for jobs and extend the list that we have currently, but i wouldn't block this graduation on that.